test/Parlot.Tests/Json/JsonParser.cs

```diff
@@ -7,7 +7,7 @@ namespace Parlot.Tests.Json
 {
     public class JsonParser
     {
-        private static readonly IParser<IJson> Json;
+        private static readonly Parlot.Fluent.Parser<IJson> Json;
```

Comment:
Would it make sense to call the raw version `Parser` and the fluent version `FluentParser` in main library to disambiguate the two?

Reply:
I actually removed the root `Parser`, so I need to remove this prefix.